backend/lucene/src/main/java/org/hibernate/search/backend/lucene/document/impl/LuceneRootDocumentBuilder.java

```diff
@@ -42,7 +42,8 @@ private List<Document> assembleDocuments(MultiTenancyStrategy multiTenancyStrate
 		// We own the document content, so we finalize it ourselves.
 		Document document = documentContent.finalizeDocument( multiTenancyStrategy, tenantId, routingKey );
 		document.add( MetadataFields.searchableMetadataField( MetadataFields.typeFieldName(), MetadataFields.TYPE_MAIN_DOCUMENT ) );
-		document.add( MetadataFields.searchableRetrievableMetadataField( MetadataFields.idFieldName(), id ) );
+		document.add( MetadataFields.searchableMetadataField( MetadataFields.idFieldName(), id ) );
+		document.add( MetadataFields.searchableRetrievableMetadataField( MetadataFields.idDocValueFieldName(), id ) );
```

Comment:
Good point 😃!
It turns out that we only used the `searchableRetrievableMetadataField` only for this ID, so I've changed it instead of adding another one